File: Python/Calculator/calculator.py
def get_number(first):
    errorText = "Please enter a number"
    if first == True:
        while True:
            try: 
                return float(input("Enter first value: "))
            except ValueError: 
                print(errorText)
    else:
        while True: 
            try:
                return float(input("Enter second value: "))
            except ValueError:
                print(errorText)
class Operators: 
    def addition():
        x = get_number(True)
        y = get_number(False)
        z = x + y
        print(f"Sum: {z}")
    def subtraction(): 
        x = get_number(True)
        y = get_number(False)
        z = x - y
        print(f"Difference: {z}")
    def multiplication():
        x = get_number(True)
        y = get_number(False)
        z = x * y
        print(f"Product: {z}")
    def division(): 
        x = get_number(True)
        y = get_number(False)
        z = x / y
        print(f"Quotient: {z}")

def main(): 
    loop = True 

    while loop == True: 
        operator = str(input("To exit type (E/e)\nEnter operator (+) (-) (*) (/) : "))

        match operator: 
            case "+": 
                Operators.addition()
            case "-": 
                Operators.subtraction()
            case "*": 
                Operators.multiplication()
            case "/": 
                Operators.division()
            case "E": 
                loop = False;
            case "e": 
                loop = False;

main()

File: C++/Calculator/calculator.cpp
#include <iostream>

using namespace std;

int main(void)
{
    bool loop = true;

    while (loop == true)
    {
        string command;
        cout << "To exit type (E/e)\nOperator (+ - * /): ";
        cin >> command;

        if (command == "+")
        {
            double x, y, z;
            cout << "Enter first value: ";
            cin >> x;

            cout << "Enter second value: ";
            cin >> y;

            z = x + y;

            cout << "Sum: " << z << endl;
        }
        else if (command == "-")
        {
            double x, y, z;
            cout << "Enter first value: ";
            cin >> x;

            cout << "Enter second value: ";
            cin >> y;

            z = x - y;

            cout << "Difference: " << z << endl;
        }
        else if (command == "*")
        {
            double x, y, z;
            cout << "Enter first value: ";
            cin >> x;

            cout << "Enter second value: ";
            cin >> y;

            z = x * y;

            cout << "Product: " << z << endl;
        }
        else if (command == "/")
        {
            double x, y, z;
            cout << "Enter first value: ";
            cin >> x;

            cout << "Enter second value: ";
            cin >> y;

            z = x / y;

            cout << "Quotient: " << z << endl;
        }
        else if (command == "E" || command == "e")
        {
            loop = false;
        }
        else
        {
            cout << "Please enter a valid command" << endl;
        }
    }

    return 0;
}